assets/js/translator.js

```diff
@@ -741,6 +764,125 @@ function translateDoc() {
     }
 }
 
+function translateLink(href) { // eslint-disable-line no-unused-vars
+    $('#webpage').val(href);
+    translateWebpage();
+}
+
+function cleanPage(html) {
```

Comment:
Move this function into `handleTranslateWebpageSuccessResponse`, it doesn't need to be in global scope.